.changes/unreleased/tinymce-TINY-10636-2024-07-22.yaml

```
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Improved
+body: Updated Stylelint to the latest version
```

```suggestion
body: Updated Stylelint to version 16.6.
```
The latest version is 16.7 as of last week.